Formula/get_iplayer.rb

```diff
@@ -0,0 +1,82 @@
+class GetIplayer < Formula
+  desc "Utility for downloading TV and radio programmes from BBC iPlayer"
+  homepage "https://github.com/get-iplayer/get_iplayer"
+  url "https://github.com/get-iplayer/get_iplayer/archive/v3.01.tar.gz"
+  sha256 "0e1e16f3706efa98893e33b1602cc00bb3d8e22e269bfc5a1a078559e4c21ce6"
+  head "https://github.com/get-iplayer/get_iplayer.git", :branch => "develop"
+
+  depends_on "atomicparsley" => :recommended
+  depends_on "ffmpeg" => :recommended
+
+  resource "IO::Socket::IP" do
+    url "https://cpan.metacpan.org/authors/id/P/PE/PEVANS/IO-Socket-IP-0.39.tar.gz"
+    sha256 "11950da7636cb786efd3bfb5891da4c820975276bce43175214391e5c32b7b96"
+  end
+
+  resource "Mojolicious" do
+    url "https://cpan.metacpan.org/authors/id/S/SR/SRI/Mojolicious-7.33.tar.gz"
+    sha256 "3bf13d60663fc5fcfb7689ba71e0b7da2d5ac818d26b39da55f469ebb3f8fb02"
+  end
+
+  def install
+    resource("IO::Socket::IP").stage do
+      system "perl", "Makefile.PL", "INSTALL_BASE=#{libexec}"
+      system "make", "install"
+    end
+
+    resource("Mojolicious").stage do
+      system "perl", "Makefile.PL", "INSTALL_BASE=#{libexec}"
+      system "make", "install"
+    end
+
+    inreplace ["get_iplayer", "get_iplayer.cgi"] do |s|
+      s.gsub!(/^(my \$version_text);/i, "\\1 = \"#{pkg_version}-homebrew\";")
+      s.gsub! "#!/usr/bin/env perl", "#!/usr/bin/perl"
+    end
+
+    man1.install "get_iplayer.1"
+
+    libexec.install "get_iplayer"
+    (bin/"get_iplayer").write_env_script \
+      "#{libexec}/get_iplayer",
+      :PERL5LIB => "#{libexec}/lib/perl5:$PERL5LIB"
+
+    libexec.install "get_iplayer.cgi"
+    (libexec/"get_iplayer.cgi").chmod 0444
+
+    (libexec/"get_iplayer_web_pvr").write <<-EOS.undent
+      #!/bin/bash
+      source "#{etc}/default/get_iplayer_web_pvr"
+      echo "Starting Web PVR Manager (Ctrl-C to stop)..."
+      screen -d -m perl "#{libexec}/get_iplayer.cgi" \
```

Comment:
If you force use of /usr/bin/perl in CLI script, you should do the same here for web application. Otherwise looks OK to me.